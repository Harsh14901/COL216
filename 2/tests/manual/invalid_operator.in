747*04584+36*+**-/-
121*+0-04+-9*08+0^+
21-1280**593*+5+-++
93-9-8+592252+%+*-+
0333627-+-860*/-+-*
73+0*1^1-025*36***-
47+20+56649%*++5+--
8614**9*715-3/-7-**
94*20--024*84^-9**-
2388*-5/42*6*-27%**
