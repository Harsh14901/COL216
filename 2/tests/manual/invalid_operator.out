[*] Executing test case - 747*04584+36*+**-/-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 121*+0-04+-9*08+0^+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 93-9-8+592252+%+*-+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 0333627-+-860*/-+-*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 73+0*1^1-025*36***-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 47+20+56649%*++5+--
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 8614**9*715-3/-7-**
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 94*20--024*84^-9**-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 2388*-5/42*6*-27%**
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
